src/EditorFeatures/Core.Wpf/Options/EditorConfigDocumentOptionsProvider.EmptyCodingConventionContext.cs

```diff
@@ -15,9 +15,16 @@ private class EmptyCodingConventionContext : ICodingConventionContext
 
             public ICodingConventionsSnapshot CurrentConventions { get; } = EmptyCodingConventionsSnapshot.Instance;
 
-#pragma warning disable CS0067
-            public event CodingConventionsChangedAsyncEventHandler CodingConventionsChangedAsync;
-#pragma warning restore CS0067
+            public event CodingConventionsChangedAsyncEventHandler CodingConventionsChangedAsync
```

Comment:
why are we doing this? you said you want to take just minimum work to make it low risk?

Reply:
This was a cleanup item. I can revert this if you want, let me know.

Reply:
shouldn't this change be different PR according to your 1 PR 1 issue principle?

Reply:
I was talking about OOP work you are doing.

Reply:
➡️ Extracted to #27267